release_notes/46-release-notes.adoc

```diff
@@ -0,0 +1,309 @@
+** Typical failure scenarios for cluster upgrader are now documented. For more information, see xref:../upgrading/upgrade-roxctl.adoc#troubleshooting-upgrader_upgrade-roxctl[Troubleshooting the cluster upgrader].
+
+Flag for diagnostic bundles added to include only database information::
+
+The `roxctl central debug download-diagnostics` command that is used to create diagnostic bundles for troubleshooting has a new flag, `--with-database-only`, to only generate database metrics in the diagnostic bundle. This is helpful for large clusters when only the database information is needed, for example, to diagnose performance issues.
+
+Additional changes::
+
+* Auto sensing within the Helm charts for detecting OpenShift clusters has been changed to depend on the `project.openshift.io/v1` `APIVersion`.
+* Sensor now stores pull secrets by secret name and registry host instead of only by registry host. This reduces delegated scanning authentication failures when multiple secrets exist for the same registry within a namespace and more closely aligns with Kubernetes secrets handling. To disable this feature and cause secrets to be stored by only registry host, set ROX_SENSOR_PULL_SECRETS_BY_NAME to `false`.
+* The endpoint `/v2/compliance/scan/configurations/reports/run` method has changed from `PUT` to `POST`.
+
+
+[id="documentation-updates-450_{context}"]
+== Documentation updates
+
+
+
+
+[id="deprecated-and-removed-features_{context}"]
+== Deprecated and removed features
+
+Some features available in earlier releases have been deprecated or removed.
+
+Deprecated functionality is still included in {product-title-short} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments.
+For the most recent list of major functionality deprecated and removed, see the following table.
+Additional information about some removed or deprecated functionality is available after the table.
+
+In the table, features are marked with the following statuses:
+
+* GA: General Availability
+* TP: Technology Preview
+* DEP: Deprecated
+* REM: Removed
+* NA: Not applicable
+
+.Deprecated and removed features tracker
+[cols="4,1,1,1",options="header"]
+|===
+|Feature |{product-title-short} 4.4 |{product-title-short} 4.5 |{product-title-short} 4.6
+
+| API token authentication for OpenShift Cluster Manager
+| GA
+| GA
+| DEP
+
+|`definitions.stackrox.io`
+|DEP
+|DEP
+|
+
+| Google Container Registry integration
+| GA
+| GA
+| DEP
+
+|Kernel support packages and driver download functionality
+|NA
+|DEP
+|DEP
+
+|Reporting of Istio vulnerabilities
+|DEP
+|DEP
+|
+
+| `rhacs-collector*` and `rhacs-collector-slim*` images
+|NA
+|DEP
+|DEP
+
+|`roxctl connectivity-map`
+|DEP
+|DEP
+|
+
+|`roxctl generate netpol`
+|DEP
+|DEP
+|
+
+|`stackrox-db` Central PVC
+|GA
+|GA
+|REM
+
+|`/v1/availableAuthProviders` endpoint
+|GA
+|DEP
+|DEP
+
+
+|`/v1/clustercves/suppress` APIs
```

Comment:
It is not completely removed but moved behind env var `ROX_VULN_MGMT_LEGACY_SNOOZE` which can be enabled if someone wants to continue using the feature. The env var is disabled by default. Same for other suppress and unsuppress endpoints for node and cluster.